lms/static/js/ccx/schedule.js

```diff
@@ -141,7 +141,7 @@ var edx = edx || {};
               return !node.hidden});
             this.$el.html(schedule_template({chapters: this.showing}));
             $('table.ccx-schedule .sequential,.vertical').hide();
-            $('table.ccx-schedule .toggle-collapse').on('click', this.toggle_collapse);
+            $('table.ccx-schedule .unit a').on('click', this.toggle_collapse);
```

Comment:
Add on click listener to parent level (href link). Because for screen readers control goes to whole link (href link)